website/templates/new_home.html

```diff
@@ -1,38 +1,64 @@
 {% extends "base_new.html" %}
-
+{% load static %}
 {% block hero %}
+
+    <script src="{% static 'js/jquery.waypoints.min.js' %}"></script>
+    <script src="{% static 'js/infinite.min.js' %}"></script>
```

Comment:
@DonnieBLT I doubt because three of the files are being used here and the feature wont run without them.
```suggestion
    <script src="{% static 'js/jquery.waypoints.min.js' %}"></script>
    <script src="{% static 'js/infinite.min.js' %}"></script>
```